### PR TITLE
Release Notes 1/02/2022-1/08/2022

### PR DESCRIPTION
**Pull request by miwithro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-01-07
+
+This release is rolling out to all regions - estimated time for completed roll out is 2022-01-17 for public cloud and 2022-01-20 for sovereign clouds.
+
+### Announcement
+
+* From Kubernetes 1.23, containerD will be the default container runtime for Windows node pools. Docker support will be deprecated in Kubernetes 1.24. You are advised to test your workloads before Docker deprecation happens by following the documentation here https://docs.microsoft.com/en-us/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview.
+* Konnectivity rollout has been halted for the rest of the year. We will continue the rollout in the new calendar year.
```
Comment (tomkerkhove) [inline CHANGELOG.md:10]: Does "the rest of the year" imply 2022 or is this a copy from 2021?

Comment (phealy) [reply CHANGELOG.md:10]: That's a copy from 2021.

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+* Bug Fixes
+  * Corrected validation that silently ignored updates to HTTP proxy settings.
+  * Fixed issue that blocked creation of 0 node nodepools.
+  * CSI driver probe timeout increased to 30s avoid driver crashes on small Windows VM sizes.
+* Behavioral Change
+  * Private Cluster now supports cross-subscription VNET for PrivateLink.
+  * In 1.21+ existing and newly created clusters, all built-in storage classes will use [CSI Driver provisioners](disk.csi.azure.com) [and](file.csi.azure.com). There are no in-tree provisioners any more(kubernetes.io/azure-disk and kubernetes.io/azure-file).
+  * CPU limits for CSI drivers have been removed.
+  * Azure CNI - won't reserve VNet IP addresses for daemonset pods using hostNetwork: true"
+* Component Updates
```
Comment (justindavies) [inline CHANGELOG.md:30]: Containerd runtime on Windows is now GA

**Review by justindavies — CHANGES_REQUESTED** [review]

**Review by justindavies — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+* Kubernetes 1.19 will be removed on 2021-01-31.
+* Starting with 1.23 AKS will follow upstream kubernetes and deprecate in-tree azure authentication which is marked for deprecation to be replaced with 'exec'. If you are using Azure CLI or Azure clients, AKS will download kubelogin for users automatically. If outside of Azure CLI, users need to download and install kubelogin in order to continue to use kubectl with AAD authentication. <https://github.com/Azure/kubelogin>
+
+### Release Notes
+
+* Features
+  * Private DNS Subzone for Private Cluster is now GA.
+  * Containerd runtime on Windows is now GA
+* Preview Features
+  * Kubenet IPv6 support has been enabled all public cloud regions. See https://aka.ms/aks/ipv6  for more details.
```
Comment (phealy) [inline CHANGELOG.md:21]:
```suggestion
  * Kubenet IPv6 support public preview has been enabled all public cloud regions. See https://aka.ms/aks/ipv6 for more details.
```

**Pull request merged**